apps/vocabulary/vocabulary.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+
```

Comment:
Please insert a package declaration here. I think it should be this:

```package gnostic.metrics.v1;```

because I'd like to "promote" this proto to a dedicated directory - "metrics" - at the top-level of the gnostic repo. There I expect that we'll add other protos for other metrics that we'll add for API specs.